boards/posix/nrf52_bsim/argparse.c

```diff
@@ -110,6 +115,16 @@ void nrfbsim_register_args(void)
 		"nosim", "", 'b',
 		(void *)&nosim, cmd_nosim_found,
 		"(debug feature) Do not connect to the phy"},
+		{ false, false, true,
+		"delay_init", "", 'b',
+		(void *)&arg.delay_init, NULL,
+		"If start_offset is used, postpone initialization and startup "
+		"until start_offset is reached (by default not set)"
```

Comment:
If we already have `start_offset`, why we do need `delay_init` too? 

Based on the changes below, setting `start_offset` without `delay_init` seems to ignore the `start_offset` value, right? 

Why can we not just have a single value, `start_offset`?

Reply:
start_offset is still used (just like before) to offset the time of the device relative to the Phy time.
delay_init is a boolean that allows the user to select if they also want to hold on running the device initialization until thy Phy has reached start_offset.
delay_init ensures we don't break backwards compatibility, as it changes when the initialiation of the test cases (and MCU boot) occurs in simulated time, as some tests may be sensitive to having this be executed earlier or later.